dashboards/google-anthos/README.md

```diff
@@ -0,0 +1,35 @@
+### Dashboards for Anthos
+
+
+|Anthos Cluster Control Plane Uptime|
+|:---------------------|
+|Filename: [anthos-cluster-control-plane-time.json](anthos-cluster-control-plane-time.json)|
+|This dashboard has 4 charts to indicate the uptime of control plane components including API Server, Scheduler, ETCD and Controller Manager.|
+
+&nbsp;
+
+|Anthos Cluster Node Status|
+|:---------------------|
+|Filename: [anthos-cluster-node-status.json](anthos-cluster-node-status.json)|
+|This dashboard has 8 charts to indicate the status of node, including node condition, CPU usage, allocatable CPU cores, CPU allocatable utilization, memory usage, allocatable memory, memory allocatable utilization and available filesystem size.|
+
+&nbsp;
+
+|Anthos Cluster Pod Status|
+|:---------------------|
+|Filename: [anthos-cluster-pod-status.json](anthos-cluster-pod-status.json)|
+|This dashboard has 11 charts to indicate the status of pod, including Number of restarts per container, Container memory usage per container, Container CPU usage per container, Network ingress (bytes) per pod, Network egress (bytes) per pod and Pod phase.|
```

Comment:
Formatting nit: make capitalization consistent everywhere. If we follow this (capitalize first letter of every chart), then Node Status above should be "include Node condition, CPU usage, Allocatable CPU cores, ..."
If we follow Node Status (everything lowercase except for acronyms like CPU), this should be "including number of restarts per container, container memory usage per container, ..."

(then make metadata.yaml match this)